Close #1282: Create new 'limited availability' classification (#1300)
jdm-contrib/jdm: CONTRIBUTING.md

  - `easy`: Sites with a simple process such as a 'delete account' button
  - `medium`: Sites that do allow account deletion but require you to perform additional steps
  - `hard`: Sites that require you to contact customer services or those that don't allow automatic or easy account deletion

>   - `only_required`: Sites that only allow you to delete your account if you live in an area with privacy rights or where otherwise required to by policy (e.g the iOS App Store). This designation is meant for sites that verify you are covered by local law or policy, such as by requiring proof of residency or only allowing deletion from an iOS app, not including sites that only link to a form and do not further verify eligibility.

Reviewer: I think this sentence is a bit too long/redundant

```suggestion
  - `only_required`: Sites that only allow you to delete your account if you live in an area with privacy rights or where otherwise required to by a policy (e.g the iOS App Store). These are only used for websites that require proof that you are covered by local law or policy and verify it..
```
